Polyhedron/demo/Polyhedron/Plugins/Mesh_3/Raw_image_dialog.cpp

```diff
@@ -1,19 +1,27 @@
 #include "Raw_image_dialog.h"
+#include <QPushButton>
 
 Raw_image_dialog::Raw_image_dialog(QWidget* parent)
-  : QDialog(parent) 
+  : QDialog(parent)
 {
   setupUi(this);
 }
 
 void Raw_image_dialog::update_image_size() {
-  label_image_size->setNum((int)image_word_size() *
-			   dim_x->value() *
-			   dim_y->value() *
-			   dim_z->value());
+  int size = (int)image_word_size() *
+      dim_x->value() *
+      dim_y->value() *
+      dim_z->value() +
+      offset->value();
+  label_image_size->setText(QString("%1 B").arg(size));
```

Comment:
`int` is a too-short type the size of an image. As soon as the image will be bigger than `2GB`, that will fail on Windows (when `int` is 32 bits). You must use `std::size_t` (the right type to code for a size of a container).